src/dotty/tools/dotc/printing/SyntaxHighlighting.scala

```diff
@@ -38,7 +39,7 @@ object SyntaxHighlighting {
   def apply(chars: Iterable[Char]): Vector[Char] = {
     var prev: Char = 0
     var remaining  = chars.toStream
-    val newBuf     = new StringBuilder
+    val newBuf     = new mutable.StringBuilder
```

Comment:
Since there is no immutable `StringBuilder`, this distinction feels slightly redundant.